Userland/Libraries/LibWeb/Internals/Internals.cpp

```diff
@@ -93,7 +93,7 @@ void Internals::send_key(HTML::HTMLElement& target, String const& key_name, WebI
 
 void Internals::commit_text()
 {
-    internals_page().handle_keydown(UIEvents::Key_Return, 0, 0);
+    internals_page().handle_keydown(UIEvents::Key_Return, 0, 0x0D);
```

Comment:
I initially added it because `should_ignore_keydown_event()` makes a decision based on code point and it was returning true for "return" key, when it is send by `internals.sendText()`. I dropped this commit because it's no longer needed after I rearranged the code.